src/Client/Pages/Authentication/ForgotPassword.razor.css

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
this file is not necessary, right?

Reply:
can be used to specialize the styles of the single component as well as in angular

Reply:
Sure, I know what it's for... but unnecessary when it's empty... just trying to keep it clean ;-)

Reply:
> Sure, I know what it's for... but unnecessary when it's empty... just trying to keep it clean ;-)

I'm a mess :D